src/course-unit/course-xblock/CourseXBlock.jsx

```diff
@@ -136,28 +139,36 @@ const CourseXBlock = ({
                   iconAs={Icon}
                 />
                 <Dropdown.Menu>
-                  <Dropdown.Item onClick={() => unitXBlockActions.handleDuplicate(id)}>
-                    {intl.formatMessage(messages.blockLabelButtonDuplicate)}
-                  </Dropdown.Item>
-                  {getConfig().ENABLE_TAGGING_TAXONOMY_PAGES && (
+                  {canDuplicate && (
+                    <Dropdown.Item onClick={() => unitXBlockActions.handleDuplicate(id)}>
+                      {intl.formatMessage(messages.blockLabelButtonDuplicate)}
+                    </Dropdown.Item>
+                  )}
+                  {canManageTags && getConfig().ENABLE_TAGGING_TAXONOMY_PAGES && (
                     <Dropdown.Item onClick={openManageTagsModal}>
                       {intl.formatMessage(messages.blockLabelButtonManageTags)}
                     </Dropdown.Item>
                   )}
-                  <Dropdown.Item>
-                    {intl.formatMessage(messages.blockLabelButtonMove)}
-                  </Dropdown.Item>
-                  {canEdit && (
+                  {canMove && (
+                    <Dropdown.Item>
+                      {intl.formatMessage(messages.blockLabelButtonMove)}
+                    </Dropdown.Item>
+                  )}
+                  {canEdit && canCopy && (
```

Comment:
[question] Why did you add `canEdit` here too? What does this selector process?

Reply:
Earlier @monteri  said that I should add `canEdit` for this button and other copy/paste functionality
@monteri what do you think?

Reply:
At the stand up we decided that we would get rid of the `canEdit`